### PR TITLE
Replace request by fetch

### PR DESCRIPTION
**Pull request by KristjanESPERANTO**
I have removed the deprecated dependency `request`. This also has the advantage that the user no longer has to perform `npm install` after cloning.

Unfortunately I can't test it because I can't get an API key. I only get the following error message when I try to register: `Please enter your business email address.`

### CODE REVIEW

**Comment by PierreGode** [comment]
Send me a hello to pierre@gode.one and I’ll send you a token.

**Review by PierreGode — APPROVED** [review]
looks good to me

File: README.md [hunk 1]
```diff
@@ -13,33 +13,29 @@ A module for the [MagicMirror²](https://magicmirror.builders) to display air qu
...
+## Update
 
-3. Navigate to the module folder:
+Just enter the module's directory and pull the update:
 
-    ```bash
-    cd MMM-airquality
-    ```
+```bash
+cd ~/MagicMirror/modules/MMM-NHL
```
Comment (KristjanESPERANTO) [inline README.md:28]: Sorry for the NHL :facepalm: But you've already fixed it :+1:

Comment (PierreGode) [reply README.md:28]: Haha no worries. Thanks for the updates 😊

**Pull request merged**